src/components/position.js

```diff
@@ -1,8 +1,8 @@
-var coordinatesMixin = require('../utils/coordinates').componentMixin;
 var registerComponent = require('../core/component').registerComponent;
-var utils = require('../utils/');
 
-module.exports.Component = registerComponent('position', utils.extend({
+module.exports.Component = registerComponent('position', {
+  type: 'coordinate',
```

Comment:
the difference is in the ```default``` reserved key. If the schema object has a ```default``` property then the component is treated as flat and the rest of the properties ignored